### ISSUE TITLE
Temporary Bookmark

### ISSUE THREAD

**Issue by dan1982code** [nvdaes/placeMarkers#2]
This is a feature request, not a bug.

The idea is to support a temporary bookmark. For example, I am reading a book and want to save my place. I could hit some keystroke, maybe windows+NVDA+k. If I move around and hit windows+NVDA+k a few times, it keeps overwriting the previous one... only one temporary bookmark. Then, there is a keystroke, windows+NVDA+shift+k or something, that jumps to the temporary bookmark that was last set. The temporary one doesn't have an associated note and doesn't show up in the bookmarks list and can't be deleted. It also doesn't show up in the NVDA+k or the NVDA+shift+k cycling.

A few design decisions here, I don't know which are ultimately best, but I do appreciate you reading and considering this request. Thanks!

**Comment by nvdaes** [comment]
Thanks for your detailed request. I'll try to implement it.

**Comment by nvdaes** [comment]
@dan1982code: Please, test this release and provide feedback:
https://github.com/nvdaes/placeMarkers/releases/download/9.0-dev-i2/placeMarkers-9.0-dev-i2.nvda-addon
